package.json

```diff
@@ -22,9 +23,12 @@
     "clean:examples:localvideofilter": "rimraf examples/localvideofilter/public/index.js examples/localvideofilter/public/helpers.js",
     "clean:examples:localvideosnapshot": "rimraf examples/localvideosnapshot/public/index.js examples/localvideosnapshot/public/helpers.js",
     "clean:examples:mediadevices": "rimraf examples/mediadevices/public/index.js examples/mediadevices/public/helpers.js",
+    "clean:examples:roomevents": "rimraf examples/roomevents/public/index.js examples/roomevents/public/helpers.js",
     "clean:examples:screenshare": "rimraf examples/screenshare/public/index.js examples/screenshare/public/helpers.js",
     "clean:quickstart": "rimraf quickstart/public/index.js",
-    "start": "npm run clean && npm run build && node server"
+    "start": "npm run clean && npm run build && node server",
+    "watch:roomevents": "copyfiles -f examples/roomevents/src/helpers.js examples/roomevents/public && watchify examples/roomevents/src/index.js -d -o examples/roomevents/public/index.js",
```

Comment:
@makarandp0 Let's not add watch events now. We can add it to all examples and quickstart in a separate PR.